aspnetcore/signalr/diagnostics.md

```diff
@@ -0,0 +1,206 @@
+---
+title: Diagnostics
```

Comment:
```suggestion
title: Diagnostics in ASP.NET Core SignalR
```

Reply:
It's already under the "Real-time apps" heading. Most of the other TOC entries don't include SignalR. It feels like it might be a bit out of place. I do think changing it in the h1 and document title makes sense.

![image](https://user-images.githubusercontent.com/7574/53526007-f700ac80-3a97-11e9-8dcb-bf58d141229c.png)

Reply:
(Also per @tdykstra 's earlier comment it's now "Logging and diagnostics")